src/ray/protobuf/gcs.proto

```diff
@@ -146,6 +146,8 @@ message ActorTableData {
   // Resource mapping ids acquired by the leased worker. This field is only set when this
   // actor already has a leased worker.
   repeated ResourceMapEntry resource_mapping = 15;
+  // Whether the local dependencies of the actor are resolved.
+  bool local_dependency_resolved = 16;
```

Comment:
this is not needed. We just need a new state.

Reply:
> this is not needed. We just need a new state.

Okey, but it will change a lot.